engine/table/src/main/java/io/deephaven/engine/table/impl/util/DynamicTableWriter.java

```diff
@@ -162,13 +164,20 @@ public Row getRowWriter() {
      * @return a RowSetter for the given column
      */
     @Override
-    public RowSetter getSetter(String name) {
+    public PermissiveRowSetter getSetter(String name) {
```

Comment:
Not relevant explicitly for this review, but I think we are planning to deprecate DynamicTableWriter eventually and push the input table interface. Hopefully we can provide nice wrappers for that soon.